python/lsst/afw/image/image/multiband.py

```diff
@@ -484,10 +484,31 @@ def removeMaskPlane(name):
 
         Parameters
         ----------
-        name: `str`
-            Name of the mask plane to remove
+        name : `str`
+           Name of the mask plane to remove
         """
-        Mask[MaskPixel].removeMaskPlaneDict()
+        Mask[MaskPixel].removeMaskPlane(name)
+
+    def removeAndClearMaskPlane(self, name, removeFromDefault=False):
+        """Remove and clear a mask from the mask plane
+
+        Clear all pixels of the specified mask and remove the plane from the
+        mask plane dictionary.  Also optionally remove the plane from the
+        default dictionary.
+
+        Parameters
+        ----------
+        name : `str`
+           Name of the mask plane to remove
+        removeFromDefault : `bool`, optional
+           Whether to remove the mask plane from the default dictionary.
+           Default is `False`.
+        """
+        # Clear all masks in MultibandMask but leave in default dict for now
+        for single in self.singles:
+            single.removeAndClearMaskPlane(name, False)
+        # Now remove from default dict according to removeFromDefault
+        self._refMask.removeAndClearMaskPlane(name, removeFromDefault)
```

Comment:
Would `self.Mask().removeAndClearMaskPlane(name, removeFromDefault)` work?

Reply:
I’m not entirely sure, but I think not as all the non-static methods this class do it this way (e.g. **clearAllMaskPlanes**, **getMaskPlaneDict**, **getNumPlanesUsed**, etc...), and **removeAndClearMaskPlane** is a non-static method: 
https://github.com/lsst/afw/blob/master/include/lsst/afw/image/Mask.h#L477

Reply:
To be sure, I ran this by @TallJimbo and he concurred.